provider/internal/device/ftd/resource.go

```diff
@@ -79,7 +79,7 @@ func (r *Resource) Schema(ctx context.Context, req resource.SchemaRequest, resp
 				},
 			},
 			"performance_tier": schema.StringAttribute{
-				MarkdownDescription: "The performance tier of the virtual FTD, if virtual is set to false, this field is ignored as performance tiers are not applicable to physical FTD devices.",
+				MarkdownDescription: "The performance tier of the virtual FTD, if virtual is set to false, this field is ignored as performance tiers are not applicable to physical FTD devices. Allow values are: \"FTDv5\", \"FTDv10\", \"FTDv20\", \"FTDv30\", \"FTDv50\", \"FTDv100\", \"FTDv\".",
```

Comment:
Allowed, not allow.  Please use the same style.

Reply:
thanks fixed